components/camera/collector.go

```diff
@@ -96,7 +95,11 @@ func newReadImageCollector(resource interface{}, params data.CollectorParams) (d
 
 		ctx = context.WithValue(ctx, data.FromDMContextKey{}, true)
 
-		img, release, err := ReadImage(ctx, camera)
+		mimeStr := new(wrapperspb.StringValue)
+		if err := mimeType.UnmarshalTo(mimeStr); err != nil {
+			return nil, err
+		}
```

Comment:
this needs to happen outside of the capture function, on line 91

Reply:
Gotcha moved